entrypoint.py

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+
+from case_utils.case_validate import validate
+from case_utils.case_validate.validate_types import ValidationResult
+
+# Get the environment variables as the settings for the validation
+case_version: str = os.environ.get("CASE_VERSION", "case-1.2.0")
```

Comment:
This seems like a bit of a deep point to bury a hard-coded version.  However, if the intent is to provide the latest ontology version provided by `case-utils`, there is a variable for that:

```python
case_utils.ontology.version_info.CURRENT_CASE_VERSION
```

Reply:
Updated